app/views/layouts/application.html.erb

```diff
@@ -6,12 +6,25 @@
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
 
+     <%= render partial: "shared/cdn_assets" %>
+
     <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
     <%= javascript_importmap_tags %>
   </head>
 
   <body>
-    
-    <%= yield %>
+    <%= render partial: "shared/navbar" %>
+
+    <div class="container">
+      <% if notice.present? %>
```

Comment:
Greta job putting your notices here so that you don't have to individually do them in every file